### ISSUE TITLE
Error: unrecognized flag --no-untrusted-code-mitigations

### ISSUE THREAD

**Issue by corwin-of-amber** [nwjs/nw.js#7807]
NWJS Version : 0.58.0, 0.58.1 (the latter built from source)
Operating System : macOS 12.0.1

### Expected behavior

Nothing in particular :grin:

### Actual behavior

This message is printed at startup:
```
Error: unrecognized flag --no-untrusted-code-mitigations
The remaining arguments were ignored: --allow_natives_syntax
Try --help for options
```

I found this line in the source repo:
https://github.com/nwjs/chromium.src/blob/aee58d2f4fc19767f4f539d9b9eff5988c500f52/content/renderer/render_process_impl.cc#L280

But did not found the corresponding flag in the V8 source tree. Is it possible that it has been removed?

### How to reproduce

Happens whenever one launches `nw`.

**Comment by MartinYounghoonKim** [comment]
Same issue here

**Comment by dlinc2** [comment]
I also have this.

**Comment by djipco** [comment]
Same here.

**Comment by rogerwang** [comment]
This should be fixed in 0.59.0.

**Comment by corwin-of-amber** [comment]
Thanks!!!
